bedrock/firefox/templates/firefox/developer-quantum-firstrun.html

```diff
@@ -68,13 +68,22 @@ <h2 class="section-title"><span>{{ _('New Tools') }}</span></h2>
         <div class="section-body">
           <h3 id="devtools" class="body-title">{{ _('Firefox DevTools') }}</h3>
 
-          <img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" alt="">
+          <a href="http://firefox-dev.tools/" rel="external"><img class="hero-image" src="{{ static('img/firefox/products/developer/quantum/hero-debugger-ani.gif') }}" {% if l10n_has_tag('quantum-firstrun-whatsnew') %}alt="{{ _('Learn more about Firefox developer tools') }}"{% endif %}></a>
```

Comment:
I think this needs an `else` to fall back to an empty alt, or just put the whole thing inside the attribute like `alt="{% if %}...{% endif %}"` so it falls back to empty.